bot/resource/cog/association/embed/asetin_embed.py

```diff
@@ -13,21 +13,21 @@ class AsetinEmbed:
 
     WHO_WE_ARE_FIELD_NAME: str = "Qui sommes-nous?"
     WHO_WE_ARE_FIELD_DESCRIPTION: str = (
-        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial."
+        "Nous sommes votre association étudiante, dédiée à offrir une variété de services aux étudiants. Nos activités incluent : \n- L'assistance aux étudiants.\n- L'organisation de concours.\n- La planification d'événements.\n- Un lieu de rencontre convivial. L'ASETIN représente les intérêts des étudiants et étudiantes de premier cycle en informatique auprès des différentes instances officielles de l’Université Laval, sur les différents comités et devant les autres associations étudiantes."
```

Comment:
I think there is a mistake here. `Un lieu de rencontre convivial. L'ASETIN représente`
You need to add a \n after the full stop.
See below:
![image](https://github.com/user-attachments/assets/3e05b5e3-3ce7-4c75-b6c8-e0be3c735786)